config/secrets.yml

```diff
@@ -0,0 +1,8 @@
+development:
+  secret_key_base: AlaveteliConfiguration.secret_key_base
```

Comment:
Err, wouldn't this literally make the value  `"AlaveteliConfiguration.secret_key_base"`.